core/server/common/src/main/webapp/metrics-summary.jsp

```diff
@@ -63,7 +69,13 @@
 			  <tr>
                 <th>Under Filesystem Read</th>
                 <th><%= request.getAttribute("totalBytesReadUfsThroughput") %></th>
-              </tr>              
+              </tr>
```

Comment:
Could you fix the formatting for the `<tr>` above as well?

Reply:
done